### PR TITLE
[New Term entry] added getDisplayNames.md

### LINKED ISSUE #2980: [Term Entry] Java Calendar getDisplayNames()
### Reason/inspiration (optional)

> **Note:** Different from the `getDisplayName()` method, there is a separate issue for `getDisplayName()` method.

Please add the `getDisplayNames()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar) class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md`

The entry should have the following:

–  An introductory paragraph about what the method is and does.
–  A syntax section with a pseudo code block with an explanation of it.
–  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ChiragAgg5k**
### Description

Added java term entry for getDisplayNames() method of the Calendar class. Closes #2980 .

### Type of Change

- New Term for Existing Concept Entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.getDisplayNames()` method of the `Calendar` class returns a map containing all string representations of Calendar field values in the given style and locale and the corresponding field values.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md:15]:
```suggestion
The `.getDisplayNames()` method of the `Calendar` class returns a map containing the string representations of the field values in the given style and locale.
```

File: content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Where:
+
+- `field`: The calendar field for which the display names are returned. For example, `Calendar.MONTH`.
+- `style`: The style applied to the display names. For example, `Calendar.LONG`.
+- `locale`: The locale for which the display names are returned. For example, `Locale.US`.
+
+It can return `null` if there are no display names for the given field and style in the given locale.
+
+Throws [NullPointerException](https://www.codecademy.com/resources/docs/java/errors/nullpointerexception) if `locale` is `null`.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md:31]:
```suggestion
The function will throw a [`NullPointerException`](https://www.codecademy.com/resources/docs/java/errors/nullpointerexception) if `locale` is `null`.
```

File: content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Example
+
+The following example, `.getDisplayNames()` method returns a map containing all days of the week in short form for the US locale. The days are paired with integer values representing the index of the day in the week (Sunday is 1, Monday is 2, etc.)
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md:35]:
```suggestion
In the following example, `.getDisplayNames()` method returns a map containing all the days of the week in short form for the US locale. The days are paired with integer values representing the index of the day in the week (Sunday is 1, Monday is 2, etc.)
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@ChiragAgg5k I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by ChiragAgg5k** [comment]
@caupolicandiaz I have made the changes.

**Review by SSwiniarski — APPROVED** [review]
@ChiragAgg5k Looks good!

**Pull request merged**